main.cpp

```diff
@@ -616,6 +745,8 @@ struct Program
     float installUpdates(float currentVersion, bool update);
     // display installed version and operating system details
     void displayProgramDetails(float installedVersion, std::string operatingSystem);
+    // new member function
+    Program startNewApplication(int threshold, int startingValue);
```

Comment:
a Program instance will return another Program?
that doesn't make much sense.
This is a member function, so the only way to call it is by using a Program instance...
- [ ] revise